lib/decidim/vocdoni/version.rb

```diff
@@ -3,10 +3,10 @@
 module Decidim
   # This holds the decidim-meetings version.
   module Vocdoni
-    DECIDIM_VERSION = "~> 0.27.1"
+    DECIDIM_VERSION = "0.27.5"
 
     def self.version
-      "0.27.1"
+      "0.27.5"
```

Comment:
```suggestion
      "1.0"
```
I'd use a custom version number (IMHO), what do you think @andreslucena ?

Reply:
Yes, let's do a custom version number + the README table with the Decidim compatibility when there are multiple compatible versions